src/renderer/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.1"
 authors = ["Eyal Kalderon <ebkalderon@gmail.com>"]
 description = "High-level rendering engine with multiple backends"
 keywords = ["game", "engine", "renderer", "3d", "amethyst"]
+categories = ["rendering"]
```

Comment:
Ah okay: https://crates.io/categories/rendering
rendering:engine = High-level solutions for rendering on the screen.
rendering:api = Crates that provide direct access to the hardware's or the operating system's rendering capabilities.

So I suppose that makes sense since we are just putting another layer onto gfx.